bot/log.py

```diff
@@ -0,0 +1,58 @@
+"""
+An implementation of a logging.Handler for sending messages to Discord
+"""
+
+import datetime
+import logging
+
+from discord import Color, Embed
+from discord.ext import commands
+
+from bot.constants import LOGGING_CHANNEL_ID
+
+LEVEL_COLORS = {
+    logging.CRITICAL: Color.red(),
+    logging.ERROR: Color.red(),
+    logging.WARNING: Color.gold(),
+    logging.INFO: Color.blurple()
+}
+
+
+class DiscordHandler(logging.Handler):
+    """
+    A class implementing logging.Handler methods to send logs to a Discord channel.
+
+    >>> handler = DiscordHandler(client) # Where client is your instance of
+    ...                                  # discord.Client or commands.Bot
+    >>> logger.addHandler(handler)
```

Comment:
It's unlikely that we'll need to use your `DiscordHandler` outside this PR, so it's probably fine to leave out these doctests.

Reply:
Right, it wasn't intended to be "tested" (it'd break anyway, no definition of client or logger). It was more to demonstrate how it would be added as a logger (& the arguments for that). I can remove now.

Reply:
Addressed in 1c8cf8307d6a10540fb4a0c7c979cf72bc07abc0